webapp/components/sidebar.jsx

```diff
@@ -280,7 +281,12 @@ export default class Sidebar extends React.Component {
 
     navigateChannelShortcut(e) {
         if (e.altKey && !e.shiftKey && (e.keyCode === Constants.KeyCodes.UP || e.keyCode === Constants.KeyCodes.DOWN)) {
+            if (this.isSwitchingChannel) {
+                return;
+            }
+
             e.preventDefault();
```

Comment:
Hmmm, wondering if `e.preventDefault();` should be above the return, otherwise if the user hits the key combination quick enough (or holds it down) it could revert to whatever their browser functionality is which could be very confusing